tests/online_migration/sle12_online_migration/zypper_patch.pm

```diff
@@ -0,0 +1,29 @@
+# SLE12 online migration tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    my $self = shift;
+    select_console 'root-console';
+
+    # fully update the system
+    zypper_call('patch --with-interactive -l');
+    zypper_call('patch --with-interactive -l', 2500);
```

Comment:
@okurz it need's double call because first updates packager/zypper itself and second rest of system.
What is missing is check for allowed return values

right way is : 

```
my $ret = zypper_call("command");
die "...." unless $ret == value .......;
```